pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/embeddings/OpenAIEmbeddingsService.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.pulsar.functions.transforms.embeddings;
+
+import com.azure.ai.openai.OpenAIClient;
+import com.azure.ai.openai.models.Embeddings;
+import com.azure.ai.openai.models.EmbeddingsOptions;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OpenAIEmbeddingsService implements EmbeddingsService {
+
+  private final OpenAIClient openAIClient;
+  private final String model;
+
+  public OpenAIEmbeddingsService(OpenAIClient openAIClient, String model) {
+    this.openAIClient = openAIClient;
+    this.model = model;
+  }
+
+  @Override
+  public List<List<Double>> computeEmbeddings(List<String> texts) {
+    EmbeddingsOptions embeddingsOptions = new EmbeddingsOptions(texts);
+    Embeddings embeddings = openAIClient.getEmbeddings(model, embeddingsOptions);
```

Comment:
The client uses the term `deploymentId` instead of `model`.
Which should we use. At least we must be consistent between Embeddings and Completion

Reply:
In the API is called "model". https://platform.openai.com/docs/api-reference/embeddings/create#embeddings/create-model
It is referred as model in the OpenAI doc: https://platform.openai.com/docs/models/embeddings

Reply:
yes but why Azure called it deploymentId then ?